src/components/ScrollableContainer.tsx

```diff
@@ -1,17 +1,26 @@
 import { DependencyList, FC, ReactNode, useEffect, useRef } from "react";
 import useEventListener from "@use-it/event-listener";
-import { getAbsoluteHeightBelow, getParentsBottomSpacing } from "util/helpers";
+import {
+  getAbsoluteHeightBelowById,
+  getAbsoluteHeightBelowBySelector,
+  getParentsBottomSpacing,
+} from "util/helpers";
+import classnames from "classnames";
 
 interface Props {
   children: ReactNode;
   dependencies: DependencyList;
-  belowId?: string;
+  belowIds?: string[];
+  belowSelectors?: string[];
+  className?: string;
 }
 
 const ScrollableContainer: FC<Props> = ({
   dependencies,
   children,
-  belowId = "",
+  belowIds = ["status-bar"],
+  belowSelectors = [],
```

Comment:
belowSelectors seem unused, can we remove them?

Reply:
sure, we can add this if it's needed in the future :+1:

Reply:
let's keep the `getAbsoluteHeightBelowBySelector` in `util/helpers.tsx` thought?

Reply:
Yes, absolutely!